src/common/paths/me.yml

```diff
@@ -12,6 +12,8 @@ get:
         application/json:
           schema:
             $ref: "../schemas/UserResponse.yml"
+    "401":
+      $ref: "../responses/AuthorizationError.yml"
```

Comment:
Seems like just about everywhere else for `401` we use the code `"../../responses/ServerError.yml"`. `AuthorizationError.yml` seems to have specific language relating to authorization tokens. Authorization tokens can be used to hit this endpoint, so I guess that's fine, although it is mostly be used for browser sessions as far as I understand it. Might be something to keep in mind if any of the generated clients/docs are acting strange.

Reply:
Thanks @wbaker85. We'll keep an eye out. For now I think we're okay with using the more specific `AuthorizationError` unless it concerns you. FYI, we're adding this so that we can distinguish the `4xx` and `5xx` errors at login so that we can implement a retry pattern with backoff for certain non-`401` responses.